paasta_tools/envoy_tools.py

```diff
@@ -217,17 +221,22 @@ def get_multiple_backends(
     :returns backends: A list of dicts representing the backends of all
                        services or the requested service
     """
-    clusters_info = retrieve_envoy_clusters(
-        envoy_host=envoy_host,
-        envoy_admin_port=envoy_admin_port,
-        envoy_admin_endpoint_format=envoy_admin_endpoint_format,
-    )
-
-    casper_endpoints = get_casper_endpoints(clusters_info)
-
     backends: DefaultDict[
         str, List[Tuple[EnvoyBackend, bool]]
     ] = collections.defaultdict(list)
+
+    try:
+        clusters_info = retrieve_envoy_clusters(
```

Comment:
we talked a little about this in slack and i'm just dumping this here so that we don't forget:

perhaps we should randomize the box we use to check backends so that we don't completely break paasta status when the deterministic way we currently use always picks the same broken box (and maybe still do this change as well? bonus if we can figure out how to plumb this back to more user-facing code and have them know that the backend list is incomplete)